syntax = "proto3";

package mathcalc;

// Define the service
service MathCalc {
    // Function invoked to send the request
    rpc sendRequest(MathRequest) returns (MathReply) {}
}

// The request message containing requested numbers
message MathRequest {
    int32 a = 1;
    int32 b = 2;
}

// The response message containing response
message MathReply {
    int32 result = 1;
}